LottieSample/src/main/kotlin/com/airbnb/lottie/samples/QRScanActivity.kt

```diff
@@ -21,11 +21,13 @@ class QRScanActivity : AppCompatActivity(), QRCodeReaderView.OnQRCodeReadListene
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_qrscan)
 
-        qrView.setQRDecodingEnabled(true)
-        qrView.setAutofocusInterval(2000L)
-        qrView.setBackCamera()
-        qrView.setOnQRCodeReadListener(this)
-        qrView.setOnClickListener { qrView.forceAutoFocus() }
+        qrView.run {
```

Comment:
make the code more concise.
reference: [http://kotlinlang.org/api/latest/jvm/stdlib/kotlin/run.html](http://kotlinlang.org/api/latest/jvm/stdlib/kotlin/run.html)